pkg/table/table.go

```diff
@@ -74,10 +75,9 @@ func AppendTemplateEbs(data [][]string, mappings []*ec2.LaunchTemplateBlockDevic
 }
 
 // Append all  EBS blocks, if applicable
-func AppendEbs(data [][]string, mappings []*ec2.BlockDeviceMapping) [][]string {
+func AppendEbs(data [][]string, mappings []*ec2.BlockDeviceMapping) ([][]string, questionModel.Row) {
```

Comment:
Discussed offline. Since this repo has ~1500 instances of initialisms not properly capitalized, we're not going to fix them in this PR. If we do so eventually, it should be as a standalone PR with an auditable process.